src/brasil/gov/vlibrasnews/tests/test_subscribers.py

```diff
@@ -40,23 +41,23 @@ def test_validate(self):
 
     def test_create_translation_ok(self):
         with HTTMock(vlibras_ok):
-            self.assertTrue(subscribers.create_translation(self.document, None))
+            self.assertTrue(subscribers.create_translation(self.document, Mock(action='publish')))
```

Comment:
my intention here was to test the method associated with the event, not to test the event trigger machinery.. for me this is sufficient to test how the method behave.